docs/recent_activity/recent_activity.md

```diff
@@ -1,70 +1,54 @@
 ---
-description: Monitor recent activity logged actions.
+description: Monitor recently logged user actions.
+edition: experience
 ---
 
-# Recent activity log [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+# Review recent user activity
```

Comment:
It's just an introduction, "action" is not needed here, in my opinion.
```suggestion
# Recent user activity
```

Reply:
I wouldn't change my section title. "user" could make reader forget that this is about repository activity and if there is a user, it could be a user logged by a scheduled script or other automation.